dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -32,7 +32,14 @@ const decideChildren = (
 	placeholderHeight?: number,
 ) => {
 	if (!clientOnly) return children; // Server side rendering
-	if (placeholderHeight) return <Placeholder height={placeholderHeight} />; // Portal using placeholder
+	if (placeholderHeight)
+		return (
+			<Placeholder
+				height={placeholderHeight}
+				shouldShimmer={false}
+				backgroundColor="white"
```

Comment:
Turn off the grey shimmer for all Islands

Reply:
Why are we turning this off?

Reply:
Because there are two loading states, the server side loader and then the loader that gets rendered by the client component, and the transition between them can be janky.

I discussed the approach of having a plain white background for both the server and client loader with @HarryFischer . He's happy to start with this and then we can look at building on top of this. For example, maybe we can render white on the server and only have a grey loader on the client?

Note. I think this should be 'transparent' not 'white', I'll fix that.